k8s/upgrades/jiva0.6-0.8.1-upgrade.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
```

Comment:
@ashishranjan738 I am a bit confused. Is this the script that should be used during the upgrade, or the script from `jiva-0.6.0-0.8.1/jiva_volume_upgrade.sh`? Looking at the script below it seems like they are two separate paths of upgrading.

Reply:
removed this file. it came by mistake.